frontend/src/routes/playground/+layout.ts

```diff
@@ -0,0 +1 @@
+export const prerender = true;
```

Comment:
I don't think we need this now that we're not using GitHub pages anymore.

Reply:
Are you sure? I think sveltekit will still pre-render this if it can and it'll be faster

Reply:
Can definitely leave it if we think it's better just done need it anymore I think.